client/ctclient/ctclient.go

```diff
@@ -251,7 +311,8 @@ func dieWithUsage(msg string) {
 		"   upload      upload cert chain and show SCT (needs -cert_chain)\n"+
 		"   getroots    show accepted roots\n"+
 		"   getentries  get log entries (needs -first and -last)\n"+
-		"   inclusion   get inclusion proof (needs -leaf_hash and optionally -size)\n")
+		"   inclusion   get inclusion proof (needs -leaf_hash and optionally -size)\n"+
+		"   consistency   get consistency proof (needs -size and -prev_size, optionally -tree_hash and -prev_hash)\n")
```

Comment:
The `command       description` don't line up anymore, can you rejustify (or perhaps sub in some \t for spaces)?